src/main/java/org/spongepowered/api/util/command/dispatcher/TranslationPlaceholder.java

```diff
@@ -31,7 +31,7 @@
 /**
  * This class contains methods intended for use as placeholders until true localization is implemented for SpongeAPI strings.
  */
-class TranslationPlaceholder {
+public class TranslationPlaceholder {
```

Comment:
please don't -- just copy the class if you need it in a different package. This is a temporary thing, needs to be replaced by a real translation system eventually.